mkt/webapps/tasks.py

```diff
@@ -715,6 +715,10 @@ def _get_trending(app_id):
         return {}
 
     def _score(week1, week3):
+        # If last week app installs are < 100, this app isn't trending.
+        if week1 < 100.0:
+            return 0.0
+
         score = 0.0
         if week3 > 1:
```

Comment:
This will work just fine, but maybe good to be consistent as float all the way through though.